Duplicati/Library/Main/Volumes/FilesetVolumeWriter.cs

```diff
@@ -150,6 +150,7 @@ public void AddControlFile(string localfile, CompressionHint hint, string filena
         public override void Dispose()
         {
             this.Close();
+            base.Dispose();

```

Comment:
This removes the temporary file containing the `filelist.json` that remained after each backup operation.  The base class contains a temporary file that is removed upon calling `Dispose`, so we need to call the `base` implementation here.